packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -3,8 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import _ from 'underscore';
-import AccountRecoveryConfirmKey from '../views/account_recovery_confirm_key';
 
+import AccountRecoveryConfirmKey from '../views/account_recovery_confirm_key';
```

Comment:
Wait, are we auto formatting/sorting imports now? Looks like the only change is that this moved down a line.